src/AddressRegistry.Api.BackOffice.Abstractions/Requests/CorrectAddressDeregularizationBackOfficeRequest.cs

```diff
@@ -0,0 +1,16 @@
+namespace AddressRegistry.Api.BackOffice.Abstractions.Requests
+{
+    using System.Runtime.Serialization;
+    using Newtonsoft.Json;
+
+    [DataContract(Name = "CorrectAddressDeregularization", Namespace = "")]
+    public class CorrectAddressDeregularizationBackOfficeRequest
```

Comment:
Can we rename to CorrectAddressDeregularizationRequest (or  CorrectAddressDeregulationRequest)? So without BackOffice.